frontend/src/components/KeyMeasureDiagnostic/ExpeModals/ExpeVegetarian.vue

```diff
@@ -0,0 +1,788 @@
+                        :value="wasteEvolutionItem.value"
+                        :key="wasteEvolutionItem.value"
+                      >
+                        <template v-slot:label>
+                          <span class="body-2 grey--text text--darken-3">{{ wasteEvolutionItem.label }}</span>
+                        </template>
+                      </v-radio>
+                    </v-radio-group>
+
+                    <!-- Gaspillage ressenti poucentage -->
+                    <label
+                      class="body-2 grey--text text--darken-3 font-weight-medium"
+                      :for="`waste-evolution-percentage-${item.value}`"
+                      v-if="!!expe[`wasteEvolution${item.value}`] && expe[`wasteEvolution${item.value}`] !== 'same'"
+                    >
+                      Écart du gaspillage en pourcentage
+                    </label>
+                    <v-text-field
+                      validate-on-blur
+                      hide-details="auto"
+                      :rules="[validators.nonNegativeOrEmpty]"
+                      solo
+                      dense
+                      v-model.number="expe[`wasteEvolutionPercentage${item.value}`]"
+                      append-icon="mdi-percent"
+                      style="max-width: 150px"
+                      class="mt-2 mb-8 body-2"
+                      :id="`waste-evolution-percentage-${item.value}`"
+                      v-if="!!expe[`wasteEvolution${item.value}`] && expe[`wasteEvolution${item.value}`] !== 'same'"
+                    ></v-text-field>
+
+                    <!-- Gaspillage -->
+                    <label class="body-2 grey--text text--darken-3 font-weight-medium">
+                      Résultats des pesées de gaspillage alimentaire sur au moins 3 jours consécutifs (en g/convive)
+                    </label>
+                    <div class="my-4" style="border-bottom: solid 1px #EEE;">
+                      <div
+                        class="d-flex py-2"
+                        style="border-top: solid 1px #EEE;"
+                        v-for="wasteItem in waste"
+                        :key="wasteItem.htmlId"
+                      >
+                        <label
+                          class="body-2 grey--text text--darken-3 pr-2"
+                          :style="$vuetify.breakpoint.mdAndUp ? 'width: 55%;' : 'width: 70%;'"
+                          :for="`${wasteItem.htmlId}-${item.value}`"
+                        >
+                          {{ wasteItem.label }}
+                        </label>
+                        <div>
+                          <v-text-field
+                            validate-on-blur
+                            hide-details="auto"
+                            solo
+                            dense
+                            v-model.number="expe[`${wasteItem.fieldName}${item.value}`]"
+                            :id="`${wasteItem.htmlId}-${item.value}`"
```

Comment:
Maybe add a "g" suffix for these g/assiette fields? https://vuetifyjs.com/en/api/v-text-field/#props-suffix

Reply:
Fix here : https://github.com/betagouv/ma-cantine/pull/1237/commits/9e5173f1687a4fa3f6d6f3b2ad5cfd7cdff0b53e